Net/SMTP.php

```diff
@@ -965,6 +966,45 @@ protected function authGSSAPI($uid, $pwd, $authz = '')
         return true;
     }
 
+    /**
+     * Authenticates the user using the XOAUTH2 method.
+     *
+     * @param string $uid   The userid to authenticate as.
+     * @param string $token The access token to authenticate with.
+     * @param string $authz The optional authorization proxy identifier.
+     *
+     * @return mixed Returns a PEAR_Error with an error message on any
+     *               kind of failure, or true on success.
+     * @since 1.8.2
```

Comment:
I might tag the next release as 1.9.0 to reflect this new feature, but I can update this as part of the release process.

Reply:
Yeah, that was just an assumption I made. Is there an ETA for the next release?

Reply:
@thomascube I just released version 1.9.0.